Firmware/ultralcd.h

```diff
@@ -133,6 +133,9 @@ extern uint8_t farm_mode;
 extern int farm_timer;
 extern uint8_t farm_status;
 
+extern bool UserECoolEnabled();
+extern bool FarmOrUserECool();
```

Comment:
I think it is misleading to have 2 functions there, someone could use wrong one.
I suggest remove FarmOrUserECool() and hide its logic inside UserECoolEnabled(). I don't know what is "User" there. Why it is not called UseECool(), or ECoolEnabled()? Effect will be, that in farm_mode hidden menu will always show ECool [On] and toggling will left it [On] but it is absolutely ok for me.

Reply:
I don't like the idea of merging the farm and user-selected (that's why I called it "User") E-cool mode